packages/fxa-settings/.storybook/main.js

```diff
@@ -4,9 +4,5 @@
 
 module.exports = {
   stories: ['./design-guide/main.stories.tsx', '../src/**/*.stories.tsx'],
-  addons: [
-    '@storybook/addon-actions',
-    '@storybook/addon-links',
-    '@storybook/preset-create-react-app',
```

Comment:
I'm not sure what this addon was doing for us, but a lot of things started working when I took it out.

Weirdly, fxa-react and fxa-payment-server weren't using it, but fxa-settings and fxa-admin-panel were. So, it resulted in a very different initial Webpack config when trying to apply the shared custom config patches.

We might want to revisit reintroducing it - but across the board rather than half-and-half. It promises to "just work" for CRA apps, so maybe it would replace our tweaks entirely at some point? 🤷‍♂️